src/core/curry.js

```diff
@@ -43,6 +43,13 @@ function curry(fn) {
     value: true
   })
 
+  Object.defineProperty(curried, 'length', {
+    configurable: true,
```

Comment:
Why allow this to be `configurable`? 
I would think that this will always be `Number` and should never be able to be deleted.

Reply:
This is a good question and I wish I had a good answer in response.

I literally took inspiration from the following in DevTools:

```
Object.getOwnPropertyDescriptors(() => {}).length
```

I'm more than happy to remove this or set it to `false`.

Reply:
I would say, remove it for now. I cannot see a need for it to be `configurable`.
It defaults to `false` so no need to explicitly set it (unless you want to communicate that this property is in debate?)

Reply:
Excellent! I've remove the property. Thanks for bringing this one up because I wasn't entirely sure whether it should be included or not 👍